src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/PushMetadataToBuildAssetRegistry.cs

```diff
@@ -474,6 +485,49 @@ internal void AddAsset(List<AssetData> assets, string assetName, string version,
             });
         }
 
+        internal (List<PackageArtifactModel>,
+            List<BlobArtifactModel>) MergeManifests(List<Manifest> parsedManifest)
```

Comment:
This is a list, so parsedManifest should be renamed to parsedManifests

Reply:
[nit] This seems to get the packages and blobs from a collection of manifests, It's not really merging any manifests is it? Probably needs a name change.

Reply:
outdated code